src/app/HealthChecks/CosmosHealthCheckDetails.cs

```diff
@@ -66,10 +67,7 @@ private HealthzCheck BuildHealthzCheck(string uri, double targetDurationMs, Exce
         /// <returns>HealthzCheck</returns>
         private async Task<HealthzCheck> GetGenresAsync(Dictionary<string, object> data = null)
         {
-            // TODO - reduce 400 to 100-250
-            // TODO - use constants
             const string name = "getGenres";
-            const int maxMilliseconds = 400;
```

Comment:
Might consider moving max ms for all methods to config file or including an enum or private class within CosmosHealthCheck and declare the maxMilliseconds needed for each method there.